tests/features/support/steps.js

```diff
@@ -223,11 +225,59 @@ When(/I check the "([^"]+)" checkbox/, async (label_text) => {
   *    Very limited. Anything more is a future feature.
   * 
   * "checkbox": label that contains checkbox-like behavior.
+  * 
+  * May switch to using the below instead - almost same code, but
+  *    its text has to match exactly and it turns out clicking labels
+  *    works for more than one thing.
   */
   let checkbox = await scope.page.waitFor( `label[aria-label*="${ label_text }"]` );
   await checkbox.click();
 });
 
+When('I pick the {string} option', async (label_text) => {
+  /* Clicks the first label "containing" the "label text".
+  *    Very limited. Anything more is a future feature.
+  */
+  let choice = await scope.page.waitForSelector( `label[aria-label="${ label_text }"]` );
+  await choice.click();
+
+  await scope.waitForShowIf(scope);
+});
+
+// TODO: Should it be 'containing', or should it be exact? Might be better to be exact.
+// TODO: Should we have a test just for the state of MA to be selected? Much easier... Or states in general
+When('I select the {string} option from the {string} choices', async (choice_text, label_text) => {
+  /* Selects the option having exactly the text `choice_text`
+  * in the <select> with the label "containing" the `label text`.
+  *    Finding one out of a bunch is a future feature.
+  * 
+  * Note: `page.select()` is the only way to click on an element in a <select>
+  */
+  // Make sure ajax is finished getting the items in the <select>
+  await scope.page.waitForSelector('option');
+
+  // The <label> will have the `id` for the <select> we're looking for
+  let [label] = await scope.page.$x(`//label[text()="${label_text}"]`);
+  let select_id = await scope.page.evaluate(( label ) => {
+    return label.getAttribute('for');
+  }, label);
+
+  // Get the actual option to pick. Can't use `value` here unfortunately as it doesn't reflect the text
+  let select = await scope.page.waitForSelector(`#${ select_id }`);
+
+  // Get the option with the exactly matching text
+  let option_value = await scope.page.evaluate(( select_elem, option_text ) => {
+    let options = select_elem.querySelectorAll( 'option' );
```

Comment:
It will be more consistent to stick to puppeteer functions. Rewrite as `await select_elem.$$( 'option' )`?

Reply:
I don't believe we can do that inside the `evaluate` function - I don't think it has access to the outside context. For example, on [line 275](https://github.com/SuffolkLITLab/docassemble-MAVirtualCourt/pull/652/files/014cb9a08c385b4850f923361d97d39cfcdc3dee#diff-b5a1ce246c4b20e9a3a17ad908d80816R275) I have to pass in `choice_text` to get it in there.

I might be able to get the list outside of the function and passing that in instead of the `select` handle, though. Not sure why I didn't do that. I wonder if I ran into a problem or if I just didn't think of it. I'll try to keep it in mind and give it a shot at some point.